Survey.xcodeproj/project.pbxproj

```diff
@@ -1474,7 +1474,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.khanh.survey;
 				PRODUCT_NAME = Survey;
 				SDKROOT = iphoneos;
-				SWIFT_ACTIVE_COMPILATION_CONDITIONS = STAGING;
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUGSTAGING;
```

Comment:
Just FYI, you can use multiple flags by adding space between them. If you want to use multiple flags in the future.
```
				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG STAGING NewFlag;
```

Reply:
Thank you for your information 💚